### PR TITLE
Update device QR scanning communications

### LINKED ISSUE #5985: Improve communication around QR code scanning in Console
### Summary

The UI around QR code scanning in the Console suggests that any kind of QR code that is on the device can be scanned to onboard the device, when it's actually only specific QR codes that adhere to a [specification set by the LoRa Alliance](https://lora-alliance.org/resource_hub/lora-alliance-vendor-id-for-qr-code/).

### Current Situation

Users are misled to believe that any type of QR code on their end device is usable for device onboarding. Some device manufacturers put arbitrary QR codes on their devices that are for internal/proprietary use only. If a user scans such QR code, it cannot be used to help onboarding the device.

### Why do we need this? Who uses it, and when?

The flow here needs to be improved to clarify that only a certain type of QR code can be used for onboarding, to save the user the trouble of (re)trying the QR scan.

### Proposed Implementation

Unfortunately, it's not really possible to discriminate working QR codes from non-working QR codes purely by appearance. So with that in mind, we can only manage expectations when scanning a QR code. I propose the following adjustments:

1. Add a text stating that only QR codes that are 'TR005 LoRaWAN Device Identification QR Codes' can be used for identifying the device, and that there can be QR codes on the physical end device that are not meant for such purpose.
2. When a QR code is scanned and found to have invalid data, this message should be repeated to let users know that they likely deal with an incompatible QR code.

### Contributing

- [X] I can help by doing more research.
- [X] I can help by implementing the feature after the proposal above is approved.
- [X] I can help by testing the feature before it's released.

### Code of Conduct

- [X] I agree to follow TTN's [Community Code of Conduct](https://github.com/TheThingsNetwork/Manifesto/blob/master/CodeOfConduct.md).

### PR DESCRIPTION
**Pull request by mjamescompton**
#### Summary

Close #5985

also see

https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/1055

#### Changes

- Update `hasEndDeviceQR` and `invalidData` communications in the device onboarding

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by ryaplots — APPROVED** [review]
Looks good to me! Only some CI is not passing

File: pkg/webui/components/qr-modal-button/index.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ const QrScanDoc = (
 const m = defineMessages({
   scanEndDevice: 'Scan end device QR code',
   scanEndDeviceContinue: 'Please scan the QR code to continue. {qrScanDoc}',
-  invalidData: 'Invalid QR data please try again',
+  invalidData:
+    'Invalid QR code data. Please scan a valid TR005 LoRaWAN® Device Identification QR Code. ',
```
Comment (kschiffer) [inline pkg/webui/components/qr-modal-button/index.js]:
```suggestion
    'Invalid QR code data. Please note that only TR005 LoRaWAN® Device Identification QR Code can be scanned. Some devices have unrelated QR codes printed on them that cannot be used.',
```

File: pkg/webui/console/containers/device-onboarding-form/messages.js [hunk 2]
```diff
@@ -63,7 +63,8 @@ export default defineMessages({
   typeToSearch: 'Type to search…',
   unknownHwOption: 'Unknown ver.',
   // QR code section.
-  hasEndDeviceQR: 'Does your end device have a QR code? Scan it to speed up onboarding.',
+  hasEndDeviceQR:
+    'Does your end device have a TR005 LoRaWAN® Device Identification QR Code? Scan it to speed up onboarding.',
```
Comment (kschiffer) [inline pkg/webui/console/containers/device-onboarding-form/messages.js]:
```suggestion
    'Does your end device have a LoRaWAN® Device Identification QR Code? Scan it to speed up onboarding.',
```
Let's be less clunky.

**Review by kschiffer — CHANGES_REQUESTED** [review]
See my comments

**Review by kschiffer — APPROVED** [review]

**Pull request merged**